subworkflows/local/longread_filtering.nf

```diff
@@ -0,0 +1,33 @@
+//
+// Check input samplesheet and get read channels
+//
+
+include { FILTLONG                   } from '../../modules/nf-core/filtlong/main'
+include { NANOQ                      } from '../../modules/nf-core/nanoq/main'
+
+workflow LONGREAD_FILTERING {
+    take:
+    reads // [ [ meta ], [ reads ] ]
+
+    main:
+    ch_versions       = Channel.empty()
+    ch_multiqc_files  = Channel.empty()
+
+    // fastp complexity filtering is activated via modules.conf in shortread_preprocessing
+    if ( params.longread_filter_tool == 'filtlong' ) {
+        ch_filtered_reads = FILTLONG ( reads.map { meta, reads -> [ meta, [], reads ] } ).reads
+        ch_versions = ch_versions.mix( FILTLONG.out.versions.first() )
+        ch_multiqc_files = ch_multiqc_files.mix( FILTLONG.out.log )
+    } else if ( params.longread_filter_tool == 'nanoq' ) {
+        ch_filtered_reads = NANOQ ( reads.map { meta, reads -> [ meta, reads ] } , 'fastq.gz' ).reads
```

Comment:
You don't need the `.map` in this line as it's the same structure as the module itself. 
```suggestion
        ch_filtered_reads = NANOQ ( reads , 'fastq.gz' ).reads
```